plugins/poll/assets/stylesheets/common/poll-breakdown.scss

```diff
@@ -1,4 +1,4 @@
-.poll-breakdown-modal {
+.poll-breakdown.modal {
```

Comment:
this regression is now an emerging pattern - quite a few modals relied on those `${name}-modal` class names 🤔

Reply:
The migration guide does say to add a class with the same format. I guess I didn't do a good job of following my own guide... 😅